src/tools/genpolicy/src/registry.rs

```diff
@@ -273,10 +265,38 @@ fn create_verity_hash(path: &str) -> Result<String> {
     }
 
     let salt = [0u8; <Sha256 as OutputSizeUser>::OutputSize::USIZE];
-    let v = verity::Verity::<Sha256>::new(size, 4096, 4096, &salt, None)?;
-    let hash = verity::traverse_file(&file, 0, false, v)?;
+    let v = verity::Verity::<Sha256>::new(size, 4096, 4096, &salt, 0)?;
+    let hash = verity::traverse_file(&mut file, 0, false, v, &mut verity::no_write)?;
     let result = format!("{:x}", hash);
     info!("dm-verity root hash: {:?}", &result);
 
     Ok(result)
 }
+
+#[cfg(target_family = "unix")]
+async fn gunzip_file(file: &Path, _output_directory: &Path) -> Result<bool> {
```

Comment:
For the future: you should consider using `flate2`, like I did here:

https://github.com/microsoft/kata-containers/pull/29/commits/498581b20b7462d641d9a069f83c62c334825318#diff-116609935bb2b68b8b4571fbd0d289ac05be0f7b90deebf3553cfd1d1b019573R225.